docs/source/integrations/databricks_development_workflow.md

```diff
@@ -0,0 +1,220 @@
+
+To set up these features, look for instructions specific to your IDE (for instance, [VS Code](https://code.visualstudio.com/docs/python/linting)).
+
+## Prerequisites
+
+- An active [Databricks deployment](https://docs.databricks.com/getting-started/index.html).
+- A [Databricks cluster](https://docs.databricks.com/clusters/configure.html) configured with a recent version (>= 11.3 is recommended) of the Databricks runtime.
+- [Conda installed](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) on your local machine.
+
+## Set up your project
+
+### Create a virtual environment and install Kedro and dbx
+
+In your local development environment, create a virtual environment for this tutorial using Conda:
+
+```bash
+conda create --name iris-databricks python=3.10
+```
+
+Once it is created, activate it:
+
+```bash
+conda activate iris-databricks
+```
+
+With your Conda environment activated, install Kedro and dbx:
+
+```bash
+pip install kedro dbx --upgrade
+```
+
+dbx is an extension of the Databricks CLI, a command-line program for interacting with Databricks without using its UI. You will use dbx to sync your project's code with Databricks.
+
+After installing dbx, you must authenticate the Databricks CLI with your [Databricks instance](https://docs.databricks.com/dev-tools/cli/index.html#set-up-authentication).
+
+```{note}
+To install and use `dbx` you must use Python >= 3.8.
+```
+
+### Create a new Kedro project
+
+Create a Kedro project with the PySpark Iris starter using the following command in your local environment:
+
+```bash
+kedro new --starter=pyspark-iris
+```
+
+Name your new project `iris-databricks` for consistency with the rest of this guide. This command creates a new Kedro project using the PySpark Iris starter template.
+
+### Create a repo on Databricks and sync your project
+
+Create a new repo on Databricks by navigating to `New` tab in the Databricks workspace UI side bar and clicking `Repo` in the drop-down menu that appears.
+
+In this guide, you will not sync your project with a remote Git provider, so uncheck `Create repo by cloning a Git repository` and enter `iris-databricks` as the name of your new repository:
+
+![Create a new repo on Databricks](../meta/images/databricks_repo_creation.png)
+
+The next step is to use dbx to sync your project to your repo. Open a new terminal instance, activate your conda environment, navigate to your project directory and start `dbx sync`:
+
+```bash
+conda activate iris-databricks
+cd <project_root>
+dbx sync repo --dest-repo iris-databricks --source . --force-include conf/local
+```
+
+This command will sync your local directory (`--source .`) with your repo (`--dest-repo iris-databricks`) on Databricks, forcing it to include the project's `conf/local` directory (`--force-include conf/local`). When started for the first time in your project directory, `dbx sync` will write output similar to the following to your terminal:
```

Comment:
`conf/local` could include credentials and that's why we never commit anything under that directory. I don't think we should be advising users to add that directory here, to me it feels like a potential security risk. What is the reason you explicitly add it?

Reply:
If the folder is missing, `session.run()` fails.

I have a feeling that `dbx sync` respects `.gitignore`, which is why it isn't included in the first place. Maybe the way around this is to engineer .gitignore so that it includes conf/local but not its contents? I can look into this, but it would mean opening a PR on our starter to fix properly.

Reply:
I think that actually it should be possible to do a succesful `kedro run` without a `conf/local` directory. That is a bigger change and probably needs some discussion though. For the purpose of this tutorial, could we suggest that users manually create a `conf/local` directory on the Databricks side?

Reply:
Yeah that is potentially the easier solution, I'll see if I can make this a clear step that readers won't skip over.